### PR TITLE
Update PubSub API reference for Lisk Service

### LINKED ISSUE #1575: Update `Publish/Subscribe API (Lisk Service)`
### Description
- The focus of the issue is to document all the **API events** for Lisk Service.
- Update `Publish/Subscribe API (Lisk Service)` page to be compatible with Lisk Service 0.7.0
- Comment out irrelevant information for now.
- Create a new issue
  - Separate general information to subscribe guide. 
  - Check for where to put the date format and other info.

### Additional ressources
https://github.com/LiskHQ/lisk-service/blob/development/docs/api/websocket_subscribe_api.md

### PR DESCRIPTION
**Pull request by Tschakki**
Removed section about the date format, because it is not relevant here and already mentioned in the HTTP API reference.

The rest of the document, I updated according to https://github.com/LiskHQ/lisk-service/blob/development/docs/api/websocket_subscribe_api.md and I think everything is relevant in this context. 
We can still discuss to add additional guides how to use the API in the future.

Closes #1575

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc [hunk 1]
```diff
@@ -19,17 +19,18 @@ Using the wrong major version might result in a broken connection and messages n
 The specification below contains numerous examples how to use the API in practice.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc]:
```suggestion
The specification below contains numerous examples of how to use the API in practice.
```

File: docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc [hunk 2]
```diff
@@ -19,17 +19,18 @@ Using the wrong major version might result in a broken connection and messages n
...
 
 == Endpoint logic
-The logic of the endpoints is derived as follows: the method naming is always based on the following pattern: `<action>.<entity>`, where the `action` is equivalent to the method type performed on the server (i.e., `update`), and `entity` is a part of the application logic, ex. `accounts`, `transactions`.
+The logic of the endpoints is derived as follows: the method naming is always based on the following pattern: `<action>.<entity>`, where
 
-== Responses
-All responses are returned in the JSON format - `application/json`.
+* the `action` is equivalent to the method type performed on the server (i.e., `update`),
+* and `entity` is a part of the application logic, ex. `accounts`, `transactions`.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc:25]:
```suggestion
* and the `entity` is a part of the application logic, ex. `accounts`, `transactions`.
```

File: docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc [hunk 3]
```diff
@@ -63,136 +60,247 @@ Updates about a newly forged block with all its data.
...
 
-=== update.forgers
-Updates the current forgers' list, so the current forger is in the first position.
-
+=== new.transactions
+Updates about included transactions within a newly generated block.
 
 .Response
-[source,json]
+[source,js]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc]:
```suggestion
[source,json]
```

File: docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc [hunk 4]
```diff
@@ -63,136 +60,247 @@ Updates about a newly forged block with all its data.
...
 
-=== update.transactions
-Updates about transactions from the last block.
-
+=== delete.transactions
+Updates about deleted transactions within a deleted block.
+This usually happens when the chain switches forks.
 
 .Response
 [source,js]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/api/lisk-service-pubsub.adoc]:
```suggestion
[source,json]
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looking good already! Just a few minor suggestions.

**Review by TalhaMaliktz — APPROVED** [review]
LGTM

**Pull request merged**